packages/frontend/src/styles/index.css

```diff
@@ -1,3 +1,16 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+html,
+body {
+  height: 100%;
+  background: linear-gradient(
```

Comment:
Are we able to refactor this linear-gradient colour for reuse?

Reply:
I will change to to tailwind. btw, do most of our UI use tailwind instead NextUI?